spec/integration/post_spec.rb

```diff
@@ -2,7 +2,8 @@
 
 RSpec.describe 'Posts', type: :feature do
   before(:each) do
-    @user = User.create(name: 'John', photo: 'www.example.com', bio: 'Life science', posts_counter: 0)
+    @user = User.create(name: 'John', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Life science',
+                        posts_counter: 0)
```

Comment:
- [ ] Kindly work on your controllers in order to fix the n+1 issue and also In your PR description include the screenshots of your app's logs before and after the fix 
![image](https://github.com/shaaibu7/Blog-app/assets/62705947/1daa9305-2f89-44a0-a9ea-7c7b66332400)

Reply:
well resolved! ✔️